applications/system-service/screenapi.h

```diff
@@ -52,22 +53,30 @@ class ScreenAPI : public APIBase {
         }
         int width, height, channels;
         auto decoded = (uint32_t*)stbi_load(path.toStdString().c_str(), &width, &height, &channels, 4);
+        int target_width = RDISPLAYWIDTH, target_height = RDISPLAYHEIGHT;
```

Comment:
Since RDISPLAYSIZE is only used here, either it should be removed, or it should be moved to device settings.

Reply:
To clarify, I do think the right solution would be to move it to device settings and rework this section.